doc/tasks/README.md

```diff
@@ -0,0 +1,550 @@
+- Der Spieler muss auf den Button mit der Antwort klicken, die er abgeben möchte.
+
+### GUI Checkboxen anhaken
+
+- Es wird eine GUI bestehend aus einem Feld für den Aufgabentext (Text oder Bild) und mehrere Buttons mit Antworten (Text oder Bild) angezeigt
+- Der Spieler muss auf die Buttons mit der Antwort klicken, die er für richtig hält.
+- Mit einem Klick aktiviert er eine nicht aktive (nicht ausgewählte) Antwort oder deaktiviert eine aktivierte Antwort.
+- Mit einem Bestätigungsbutton muss der Spieler seine Antworten einloggen.
+
+### GUI Text eingeben
+
+- Es wird eine GUI bestehend aus einem Feld für den Aufgabentext (Text oder Bild) und einem Feld zur Texteingabe angezeigt.
+- Der Spieler muss seine Antwort in das Eingabefeld schreiben.
+- Mit einem Bestätigungsbutton muss der Spieler seine Antworten einloggen.
+
+
+### Item an NPC abgeben
+
+- Eine Menge an Items ist im Dungeon verteilt (dabei ist nicht relevant, wie und wo der Spieler die
+  Items findet und erhält)
+- Die Items können vom Spieler aufgesammelt werden
+- Bei Interaktion mit einem NPC kann der Spieler ein aufgesammeltes Item auswählen und an den NPC abgeben
+  - Hierfür öffnet sich ein UI-Element, welches das Spielerinventar anzeigt und eine Auswahl der Item(s) ermöglicht
+  - Hierbei werden Informationen über die Items angezeigt
+- Überprüfung der Korrektheit des abgegebenen Items
+- Der NPC reagiert in bestimmter Form auf den Fall, dass das falsche Item abgegeben wurde und in anderer Form
+  auf den Fall, dass das korrekte Item abgegeben wurde
+
+### Item(s) in Container ablegen
+
+- Eine Menge an Items ist im Dungeon verteilt (dabei ist nicht relevant, wie und wo der Spieler die
+  Items findet und erhält)
+- Die Items können vom Spieler aufgesammelt werden
+- Bei Interaktion mit Container kann der Spieler eine beliebige Anzahl der gesammelten Items in den Container
+  legen
+  - Hierfür öffnet sich ein UI-Element, welches das Spielerinventar anzeigt und eine Auswahl der Item(s) ermöglicht
+  - Hierbei werden Informationen über die Items angezeigt
+- Die Überprüfung der Korrektheit der im Container abgelegten Items kann zu unterschiedlichen Zeitpunkten geschehen
+  - Direkt nach dem Schließen des UI-Elements zur Auswahl der Items
+  - Bei externem Event (bspw. Betätigung eines Schalters)
+
+### Item(s) aus Container nehmen
+
+- In einem Container ist eine Menge von Items gespeichert
+- Durch Interaktion mit dem Container kann der Spieler eine beliebige Anzahl Items aus dem Container herausnehmen
+  - Hierzu öffnet sich ein UI-Element, welches das Inventar des Containers anzeigt
+     und die Auswahl der herauszunehmenden Items ermöglicht
+  - Hierbei werden Informationen über die Items angezeigt
+- Die Überprüfung der Korrektheit der aus dem Container entnommenen Items kann zu unterschiedlichen Zeitpunkten geschehen
+  - Direkt nach dem Schließen des UI-Elements zur Auswahl der Items
+  - Bei externem Event (bspw. Betätigung eines Schalters)
+
+### Blöcke schieben
```

Comment:
Ich würde vorschlagen, aus "Blöcke" ein allgemeineres "Items" oder "Entitäten" zu machen? Es ist mir ja erstmal egal, ob ich Blöcke schiebe oder ein Monster durch die Gegend schubse oder ein Nahrungsitem?

Reply:
"Entität" ist generell der Begriff für alles, was sich im Dungeon befinden kann, das ist der Begriff kommt direkt aus "Entity-Component-System", was ja die Basis für die Verwaltung aller "Dinge" im Dungeon ist. "Item" hängt (für mich) konzeptionell mit dem Inventar zusammen (ich kann ein Item aufheben, ich kann es in mein Inventar packen und es benutzen, etc.)

Reply:
@malt-r kann das zu?